src/main/java/com/dyescape/dyescapebot/constant/Config.java

```diff
@@ -0,0 +1,19 @@
+package com.dyescape.dyescapebot.constant;
+
+/**
+ * Class containing constants of all base configuration options
+ * that this application makes use of. Note that different platform
+ * implementations of this bot may have their own set of configuration
+ * options. Those platform specific options are not included in this
+ * class.
+ * @author Dennis van der Veeke
+ * @since 0.1.0
+ */
+public final class Config {
+
+    // -------------------------------------------- //
+    // CONSTANTS
+    // -------------------------------------------- //
+
+    public static final String API_TOKEN = "API_TOKEN";
```

Comment:
If the API token is defined here, what purpose does config.json serve? I'm all for caching values, but having a Config _constant_ for something that I presume will be read from a file seems nonsensical. It would make more sense if the api token were read from file and held as a private field in the bot's main acting class, `DyescapeBot.java`

Reply:
This constant file only contains the names of the config values. It doesn't directly load anything. The configuration works with a simple JSON file or environment properties. To keep stuff a bit clean, I figure'd it would be a good idea to simply define the variable names here. The Vertx ConfigurationRetriever will take care of loading the values from a config file or environment variable.